armi/reactor/converters/uniformMesh.py

```diff
@@ -65,6 +65,11 @@
 from armi.reactor.flags import Flags
 from armi.reactor.converters.geometryConverters import GeometryConverter
 from armi.reactor import parameters
+from armi.reactor import reactors
+from armi.reactor import blueprints
+from armi import operators
```

Comment:
I tested this and these imports need to be moved into `initNewReactor` because of circular imports.